src/test/java/com/epam/tc/hw4/SeleniumBaseClass.java

```diff
@@ -0,0 +1,41 @@
+package com.epam.tc.hw4;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import java.time.Duration;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.ITestContext;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.BeforeSuite;
+
+public abstract class SeleniumBaseClass {
+
+    public static final long TIMEOUT_SECONDS = 10;
+    protected WebDriver driver;
+    protected WebDriverWait wait;
+    protected ChromeOptions options;
+
+    @BeforeMethod
+    public void setUp(Object[] data, ITestContext context) {
+
+        WebDriverManager.chromedriver().setup();
+        options = new ChromeOptions();
+        options.addArguments("start-maximized");
+        options.addArguments("--disable-dev-shm-usage");
```

Comment:
`ChromeOptions` support fluent style, `.addarguments` takes vararg. So lets rewrite code to make it clean
```suggestion
        ChromeOptions options = new ChromeOptions()
                                    .addArguments("start-maximized",
                                                  "--disable-dev-shm-usage");
```